### PR TITLE
Require OSI license and suggest EUPL

### LINKED ISSUE #104: Suggest software licenses
* Take stock of licenses currently being used by CLARIAH partners.
* Add a “The software SHOULD be licensed one of the … licenses clause: one of [popular licenses](https://opensource.org/licenses) EUPL.”

Ref #102.

### PR DESCRIPTION
**Pull request by ddeboer**
Fix #104.

### CODE REVIEW

File: requirements/software-requirements.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ with the licenses of the dependencies they use. Tools such as the [JLA -
 Compatibility Checker](https://joinup.ec.europa.eu/collection/eupl/solution/joinup-licensing-assistant/jla-compatibility-checker)
 can help identify license conflicts.
 
-See the list of [OSI-approved licenses](https://opensource.org/licenses). It is *RECOMMENDED* to pick one of the licenses mentioned under *popular licenses* there.
+The license *MUST* be one of the [OSI-approved licenses](https://opensource.org/licenses).
```
Comment (ddeboer) [inline requirements/software-requirements.md:56]: Do you agree we can require this?

File: requirements/software-requirements.md [hunk 2]
```diff
@@ -53,7 +53,8 @@ with the licenses of the dependencies they use. Tools such as the [JLA -
...
+The license *SHOULD* be one of the popular licenses mentioned there or [EUPL-1.2](https://opensource.org/licenses/EUPL-1.2).
```
Comment (ddeboer) [inline requirements/software-requirements.md:57]: While EUPL-1.2 may seem a bit random, there’s two good reasons for suggesting it specifically:

1. CLARIAH is a European collaboration, so it makes sense to suggest a license that works well with European legislation;
2. This was a reason for NDE to adopt EUPL.

File: requirements/software-requirements.md (hunk 1 above)
Comment (proycon) [reply requirements/software-requirements.md:56]: Yes, sounds better indeed!

File: requirements/software-requirements.md (hunk 2 above)
Comment (proycon) [reply requirements/software-requirements.md:57]: Makes sense to me yes, agreed

**Pull request merged**